packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -164,23 +166,33 @@ const SlackErrorContent: FC<{ slackState: SlackStates }> = ({ slackState }) => {
     return <></>;
 };
 
-const SchedulerForm: FC<{
+type Props = {
     disabled: boolean;
     savedSchedulerData?: SchedulerAndTargets;
+    resource?: {
+        uuid: string;
+        type: 'chart' | 'dashboard';
+    };
     onSubmit: (data: any) => void;
     onSendNow: (data: CreateSchedulerAndTargetsWithoutIds) => void;
     onBack?: () => void;
     loading?: boolean;
     confirmText?: string;
-}> = ({
+};
+
+const SchedulerForm: FC<Props> = ({
     disabled,
+    resource,
     savedSchedulerData,
     onSubmit,
     onSendNow,
     onBack,
     loading,
     confirmText,
 }) => {
+    const isSchedulerFiltersEnabled =
+        useFeatureFlagEnabled('scheduler-filters');
```

Comment:
am I doing this right? do I need to change anything else?

Reply:
that's great!